### DISCUSSION TITLE
Looking for old SeRQL User Manual

### DISCUSSION THREAD

**Discussion by celiaudi** [eclipse-rdf4j/rdf4j#3258] [Q&A] [answered]
Hi, yes, I know SeRQL is not supported anymore, but we have a legacy system based on Sesame and have to maintain this up. We have performance issues that I can't understand, so I'd like to see this manual in order to check query examples. Have to say authors did a good job erasing footprints : ). Thanx for any help.

**Comment by hmottestad** [comment, marked as answer]
Maybe this?

http://archive.rdf4j.org/users/ch11.html

**Reply by celiaudi** [reply]
Ohhh! Thank you so much, @hmottestad

**Reply by abrokenjester** [reply]
As an aside, if you're willing to share I'd be really interested to hear a little more about this legacy system, and if the decision to stick with old Sesame + SeRQL is purely a "no resources to do the upgrade work" thing or if you're facing other issues. Unless you do very bespoke/exotic things with Sesame, an upgrade should not be a _huge_ amount of work (mainly just fixing imports), and you get a lot of benefits (including much better query performance, especially if you switch to SPARQL). There's a fairly comprehensive [guide to migrating from Sesame to RDF4J](https://rdf4j.org/documentation/reference/migration/) that covers the basics.

**Reply by celiaudi** [reply]
Thank you, Jeen. This system supports the metadata of our site at dieese.org.br - nothing exotic. As you've correctly suspected, it's pretty much a matter of "no resources to do (or had done) the upgrade work". But, to tell you the truth, I'm the only person who keeps "defending" the upgrade (we´re a small team and I´m not a developer). We've had an ambicious project 15 years ago in terms of generalizing and integrating metadata of our applications, but things didn't happen as planned. Now it's like a lost battle and probably we wiil change back to a more conventional way, which is rational considering the size of our site - I don't know yet. But I'd like to thank the persons who allow this beautiful project, anyway. Let's see how it develops, thanks to you.
